### PR TITLE
feat(StatusCommunityTags): Add community tags component

### PR DESCRIPTION
**Pull request by MishkaRogachev**
For https://github.com/status-im/status-desktop/issues/5680

Screenshot 2022-06-14 at 15 22 32

### Checklist

- [x] follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
  - the scope should be the component's name e.g: `feat(StatusListItem): ... `
  - when adding new components, the scope is the module e.g: `feat(StatusQ.Controls): ...`
- [ ] add documentation if necessary (new component, new feature)
- [x] update sandbox app
  - in case of new component, add new component page
  - in case of new features, add variation to existing component page
  - nice to have: add it to the demo application as well
- [x] test changes in both light and dark theme?
- [ ] is this a breaking change?
    - [ ] use the dedicated `BREAKING CHANGE` commit message section
    - [ ] resolve breaking changes in [status-desktop](https://github.com/status-im/status-desktop)
        - [ ] (pre-merge) adapt code to breaking changes
        - [ ] (post-merge) update StatusQ submodule pointer
- [ ] test changes in [status-desktop](https://github.com/status-im/status-desktop)

### CODE REVIEW

File: sandbox/pages/StatusCommunityTagsPage.qml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+                   "👩‍💻 Software dev",
+                   "⚽️ Sports",
+                   "📱 Tech",
+                   "🗺 Travel",
+                   "🚕 Vehicles",
+                   "🌐 Web3"
+                ]
+
+                delegate: StatusFlatButton {
+                    size: StatusBaseButton.Size.Tiny
```
Comment (PascalPrecht) [inline sandbox/pages/StatusCommunityTagsPage.qml:79]: I didn't know we had a tiny variant - nice!

If we want to have an actual `StatusCommunityTag` component we can introduce it using this definition of `StatusFlatButton` here

Comment (MishkaRogachev) [reply sandbox/pages/StatusCommunityTagsPage.qml:79]: Ok, i'll do it

**Review by noeliaSD — COMMENTED** [review]
Great job! It looks nice!!

Just some minor visual things (super minor :) ):
- Looking at the text, I have the feeling that the bottom margin is smaller than the top, only 1 or 2 pixels of difference. Maybe it's because the tag's height is also less than in figma?
- Also the space between the emoji and the text seems smaller than in figma (1 or 2 pixels difference again)
- On the sandbox page, it seems strange to me, when searching, the search input field jumps. Could the searcher input be fixed in a specific position and what expands or collapses is just the tags themselves?

**Comment by PascalPrecht** [comment]
When @noeliaSD approves this, this can be merged.

**Review by noeliaSD — APPROVED** [review]
Perfect!!

**Pull request merged**